db/migrate/20150912205745_create_seed_trades.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSeedTrades < ActiveRecord::Migration
+  def change
+    create_table :seed_trades do |t|
+      t.text       :message
+      t.text       :address
+      t.datetime   :requested_date
+      t.datetime   :accepted_date
```

Comment:
Marking as "needs: more work" until you have time to get to this, @CloCkWeRX. Otherwise, looks good!